apps/src/blockly/themes/accessibleColors.mjs

```diff
@@ -58,12 +63,22 @@ var accessiblePalette = {
   color_FF4235: '#FF4235'
 };
 
+console.log(
+  '\x1b[33m%s\x1b[0m',
```

Comment:
This just tells terminal to use yellow text to better stand out against what is being copied.
https://stackoverflow.com/questions/9781218/how-to-change-node-jss-console-font-color